kore/src/Kore/Exec.hs

```diff
@@ -32,8 +32,8 @@ import qualified Control.Monad as Monad
 import Control.Monad.Catch
     ( MonadCatch
     )
-import Control.Monad.IO.Unlift
-    ( MonadUnliftIO
+import Control.Monad.IO.Class
+    ( MonadIO
```

Comment:
Let's add (in a separate pull request) `MonadIO` to `Prelude.Kore`.

Reply:
I'll do that. But what do you mean by removing the dependency on `unliftio`?

Reply:
I mean [removing the dependency](https://github.com/kframework/kore/blob/master/kore/package.yaml#L83) from the package. Is it still used anywhere else? It's OK if it is, I just wondered if you know.